components/cam/cime_config/config_component.xml

```diff
@@ -250,11 +282,24 @@
       <value compset="_CAM5%AQUA"     >aquaplanet_EAMv1</value>
       <!-- Radiative-Convective Equilibrium (RCE) -->
       <value compset="_CAM5%RCE"      >RCEMIP_EAMv1</value>
+      <!-- MMF / Super-Parameterization -->
+      <value compset="1850_CAM5%SP1"     >1850_cam5_av1c-04p2-MMF-1mom</value>
```

Comment:
I think the 1850 and 20TR lines should go away. They do not have corresponding compset definitions in config_compsets.xml, and we are not using or testing them.

Reply:
ok